howto/how-to-set-up-a-blaster.md

```diff
@@ -0,0 +1,331 @@
+---
+title: How to set up a blaster
+---
+
+# Introduction
+If you want to setup a blaster, first you have to understand the general mechanics of the `blaster.h` prop. A blaster is always on (unless a dedicated Power button is added). The `blaster.h` prop offers three "modes": Stunt, Kill and Auto. Also, you have a certain number of shots until you need to reload. And you have a certain chance of your weapon randombly jamming.
```

Comment:
What controls if a blaster is always on or not is actually if the font contains a poweron*.wav files.  If there is no poweron file, then the saber turns on automatically.  If there is a poweronfile, then you need a power button to turn it on.